### PR TITLE
Extend migration tasks configs

### PR DESCRIPTION
**Pull request by gabrielfs7**
Extend migration tasks configs

Ticket: https://oat-sa.atlassian.net/browse/CON-256

- [ ] https://github.com/oat-sa/extension-tao-advanced-search/pull/1
- [ ] https://github.com/oat-sa/generis/pull/866
- [ ] https://github.com/oat-sa/extension-tao-outcomeui/pull/361

### CODE REVIEW

**Review by mike-ionut-mihai-sandu-tao — APPROVED** [review]
- [x] New code is covered by tests (if applicable)
- [x] Tests are running successfully (old and new ones) on my local machine (if applicable)
- [x] New code is respecting code style rules
- [x] New code is respecting best practices
- [ ] New code is not subject to concurrency issues (if applicable)
- [x] Feature is working correctly on my local machine (if applicable)
- [x] Acceptance criteria are respected
- [x] Pull request title and description are meaningful

**Review by bartlomiejmarszal — APPROVED** [review]
- [x] New code is covered by tests (if applicable)
- [x] Tests are running successfully (old and new ones) on my local machine (if applicable)
- [x] New code is respecting code style rules
- [x] New code is respecting best practices
- [ ] New code is not subject to concurrency issues (if applicable)
- [x] Feature is working correctly on my local machine (if applicable)
- [x] Acceptance criteria are respected
- [ ] Pull request title and description are meaningful

File: test/unit/models/classes/task/migration/service/ResultFilterFactoryTest.php [hunk 1]
```diff
@@ -0,0 +1,84 @@
...
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program; if not, write to the Free Software
+ * Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
+ *
+ * Copyright (c) 2020 (original work) Open Assessment Technologies SA;
```
Comment (boajer) [inline test/unit/models/classes/task/migration/service/ResultFilterFactoryTest.php:18]:
```suggestion
 * Copyright (c) 2021 (original work) Open Assessment Technologies SA;
```

File: test/unit/models/classes/task/migration/service/SpawnMigrationConfigServiceTest.php [hunk 2]
```diff
@@ -0,0 +1,91 @@
...
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program; if not, write to the Free Software
+ * Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
+ *
+ * Copyright (c) 2020 (original work) Open Assessment Technologies SA;
```
Comment (boajer) [inline test/unit/models/classes/task/migration/service/SpawnMigrationConfigServiceTest.php:18]:
```suggestion
 * Copyright (c) 2021 (original work) Open Assessment Technologies SA;
```

**Review by boajer — APPROVED** [review]
- [ ] New code is covered by tests (if applicable)
- [ ] Tests are running successfully (old and new ones) on my local machine (if applicable)
- [ ] New code is respecting code style rules
- [ ] New code is respecting best practices
- [ ] New code is not subject to concurrency issues (if applicable)
- [ ] Feature is working correctly on my local machine (if applicable)
- [ ] Acceptance criteria are respected
- [ ] Pull request title and description are meaningful

File: test/unit/models/classes/task/migration/service/SpawnMigrationConfigServiceTest.php (hunk 2 above)
Comment (gabrielfs7) [reply test/unit/models/classes/task/migration/service/SpawnMigrationConfigServiceTest.php]: done

File: test/unit/models/classes/task/migration/service/ResultFilterFactoryTest.php (hunk 1 above)
Comment (gabrielfs7) [reply test/unit/models/classes/task/migration/service/ResultFilterFactoryTest.php]: done

**Pull request merged**